docs/install.md

```diff
@@ -10,13 +10,23 @@ Kube-OVN includes two parts:
 - OS: CentOS 7/8, Ubuntu 16.04/18.04 
 - Other Linux distributions with geneve, openvswitch and ip_tables module installed. You can use commands  `modinfo geneve`, `modinfo openvswitch` and `modinfo ip_tables` to verify
 - Kernel boot with `ipv6.disable=0`
-- Kube-proxy *MUST* be ready so that Kube-OVN can connect to apiserver
+- Kube-proxy *MUST* be ready so that Kube-OVN can connect to apiserver by service address
 
 *NOTE*
 1. Users using Ubuntu 16.04 should build the OVS kernel module and replace the built-in one to avoid kernel NAT issues.
 2. CentOS users should make sure kernel version is greater than 3.10.0-898 to avoid a kernel conntrack bug, see [here](https://bugs.launchpad.net/neutron/+bug/1776778).
 3. Kernel must boot with IPv6 enabled, otherwise geneve tunnel will not be established due to a kernel bug, see [here](https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1794232).
 
+*Ports that Kube-OVN uses:*
+
+| Component           | Port                                          | Usage                  |
+|---------------------|-----------------------------------------------|------------------------|
+| ovn-central         | 6641/tcp, 6642/tcp, 6643/tcp, 6644/tcp        | ovn-db and raft server |
+| ovs-ovn             | Geneve 6081/udp, STT 7471/tcp, Vxlan 4789/udp | Tunnel port            |
+| kube-ovn-controller | 10660/tcp                                     | Metrics                |
+| kube-ovn-daemon     | 10665/tcp                                     | Metrics                |
+| kube-ovn-monitor    | 10661/tcp                                     | Metrics                |
```

Comment:
kube-ovn-pinger port

Reply:
kube-ovn-pinger uses container network and will not use host ports